packages/tfm-pulpcore/tfm-pulpcore.spec

```diff
@@ -0,0 +1,184 @@
+Requires: %{scl_prefix_python}python-setuptools
+
+%description python3-setuptools
+Empty package pulling in %{scl_prefix_python}python-setuptools
+This package is only required to ease definition of dependencies inside the SCL
+
+%prep
+%setup -T -c
+
+# This section generates README file from a template and creates man page
+# from that file, expanding RPM macros in the template file.
+cat >README <<'EOF'
+%{expand:%(cat %{SOURCE1})}
+EOF
+
+# copy the license file so %%files section sees it
+cp %{SOURCE2} .
+
+%build
+# generate a helper script that will be used by help2man
+cat >h2m_helper <<'EOF'
+#!/bin/bash
+[ "$1" == "--version" ] && echo "%{scl_name} %{version} Software Collection" || cat README
+EOF
+chmod a+x h2m_helper
+
+# generate the man page
+help2man -N --section 7 ./h2m_helper -o %{scl_name}.7
+
+%install
+rm -rf %{buildroot}
+mkdir -p %{buildroot}%{_scl_scripts}/root
+mkdir -p %{buildroot}%{_root_prefix}/lib/rpm/redhat
+cat >> %{buildroot}%{_scl_scripts}/enable << EOF
+. scl_source enable %{scl_python}
+export PATH=%{_bindir}\${PATH:+:\${PATH}}
+export LD_LIBRARY_PATH=%{_libdir}\${LD_LIBRARY_PATH:+:\${LD_LIBRARY_PATH}}
+export MANPATH=%{_mandir}:\${MANPATH}
+export PYTHONPATH="%{pulpcore_sitelib}:%{pulpcore_sitearch}:\${PYTHONPATH:+:\${PYTHONPATH}}"
+EOF
+%scl_install
+
+mkdir -p %{buildroot}%{pulpcore_sitelib}
+mkdir -p %{buildroot}%{pulpcore_sitearch}
+
+# additional rpm macros for builds in the collection to set the vendor correctly
+cat >> %{buildroot}%{_root_sysconfdir}/rpm/macros.%{scl}-config << EOF
+%%scl_vendor %{scl_vendor}
+%%_scl_prefix %{_scl_prefix}
+%%scl_package_override() %%{expand:%{?python38_os_install_post:%global __os_install_post %python38_os_install_post}
+%%global __python_requires %%python38_python_requires
+%%global __python_provides %%python38_python_provides
+# macros commonly used in specfiles
+%%global __python3 %%python38__python3
+%%global __python %%python38__python3
+%%global python3_version %%python38python3_version
+%%global python3_version_nodots %%python38python3_version_nodots
+%%global python3_platform %%python38python3_platform
+%%global py3dir %%python38py3dir
+%%global py3_build %%python38py3_build
+%%global py3_build_egg %%python38py3_build_egg
+%%global py3_build_wheel %%python38py3_build_wheel
+%%global py3_install_egg %%python38py3_install_egg
+%%global py3_install_wheel() %%%%{expand:%%{python38py3_install_wheel_start}%%%%{1}%%{python38py3_install_wheel_end}}
+%%global python3_sitelib %pulpcore_sitelib
+%%global python3_sitearch %pulpcore_sitearch
+%%global pulpcorepy3_install() %%{expand:\\\\\\
+  %%{?scl:scl enable rh-python38 '}\\\\\\
+  CFLAGS="\${CFLAGS:-\${RPM_OPT_FLAGS}}" LDFLAGS="\${LDFLAGS:-\${RPM_LD_FLAGS}}"\\\\\\
+  %%{python38__python3} %%{python38py_setup} %%{?py_setup_args} install -O1 --skip-build --root %%{buildroot} --install-purelib %%{python3_sitelib} --install-platlib %%{python3_sitearch} --install-scripts %%{_bindir} %%{?*}\\\\\\
```

Comment:
I had to add `--install-purelib %%{python3_sitelib} --install-platlib %%{python3_sitearch} --install-scripts %%{_bindir}` here, as otherwise stuff would land in the respective `rh-python38` folders.

For the curious reader: I *wanted* to just add that to the `%python38py3_install` macro, but failed as the following didn't work:
```
 %%global py3_install %%python38py3_install --install-purelib …
```
It complained that `-` is not a valid param for the macro, which is true, I wanted to add it to the `%{?*}` that the original macro has… But how?